modules/updating-eus-to-eus-upgrade-console.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * updating/preparing-eus-eus-upgrade.adoc
+
+:_content-type: PROCEDURE
+[id="updating-eus-to-eus-upgrade-console_{context}"]
+= EUS-to-EUS upgrade using the web console
+
+.Prerequisites
+
+* Have access to the web console as a user with `admin` privileges.
+
+.Procedure
+
+. Upgrade any OLM Operators to versions that are compatible with both versions you are updating to. You can find these steps on the link:https://docs.openshift.com/container-platform/4.11/operators/admin/olm-upgrading-operators.html#olm-changing-update-channel_olm-upgrading-operators[Preparing for an Operator update] page.
+
+. Verify that all machine config pools display a status of `UPDATED` and that no machine config pool displays a status of `UPDATING`.
++
+To view the status of all machine config pools, click *Compute* -> *MachineConfigPools* and review the contents of the *Update status* column.
++
+[NOTE]
+====
+If your machine config pools have an `Updating` status, please wait for this status to change to `Updated`. This process could take several minutes.
+====
+
+. Set your channel to `eus-4.12`. 
++
+To set your channel, click *Administration* -> *Cluster settings* -> *Channel*. You can edit your channel by clicking on the current hyperlinked channel.
+
+. Pause the worker machine pools that you want to skip reboots on. You can perform this action on the *MachineConfigPools* tab under the *Compute* page. Select the vertical ellipses next to the machine config pool you'd like to pause and click *Pause updates*.
+
+. Upgrade to version 4.11 by completing steps 1-3 on the link:https://docs.openshift.com/container-platform/4.12/updating/updating-cluster-within-minor.html#update-upgrading-web_updating-cluster-within-minor[Updating a cluster by using the web console] page. 
+
+. Ensure that the 4.11 upgrades are complete by reviewing the cluster version on the *Cluster settings* page under the *Details* tab.
```

Comment:
Not sure which part shall we check for complete upgrade, attaching a screenshot for reference however
![Screenshot 2023-03-08 at 1 41 40 PM](https://user-images.githubusercontent.com/12692381/223628973-886e706b-8368-45a9-947c-47637c27db1a.png)

Reply:
sounds good to me